packages/web/src/experience/eventkit/content/circles/overview.md

```diff
@@ -0,0 +1,39 @@
+# Celo Ambassador
```

Comment:
I'm skipping the markdown content, I assume experience circle will review it, or already has.